DigitalLearningSolutions.Data/Models/Register/DelegateRegistrationModel.cs

```diff
@@ -22,6 +22,7 @@ public class DelegateRegistrationModel : RegistrationModel
             string? answer6,
             bool isSelfRegistered,
             bool active,
+            bool activeUser,

```

Comment:
I have:
* renamed the variables, 
* renamed the class property `ActiveUser` to `UserIsActive`

Since `DelegateRegistrationModel` inherits from `RegistrationModel` which has a class property `Active` and is also used to register admins, I have left `DelegateRegistrationModel.Active` alone. 

Is this ok?

Reply:
Hmm, what about calling it `centreAccountIsActive`?

Reply:
Yeah, that works. Done.